docs/computing/connecting/ssh-unix.md

```diff
@@ -0,0 +1,95 @@
+# SSH client on macOS and Linux
+
+On Unix-based systems like macOS and Linux, it is recommended to connect to CSC
+supercomputers using the pre-installed terminal program.
+
+## Basic usage
+
+The OpenSSH client typically comes pre-installed on macOS and Linux systems.
+You can create a remote SSH connection by opening the terminal and running:
+
+```bash
+# Replace <username> with the name of your CSC user account and
+# <hostname> with "puhti" or "mahti"
+
+ssh <username>@<hostname>.csc.fi
+```
+
+## Graphical connection
+
+Displaying graphics, such as GUIs and plots, over an SSH connection requires
+a window system. Most macOS and Linux systems have a server program for the X
+window system (X11) installed by default.
+
+To enable displaying graphics over SSH, use the `-X` (X11 forwarding) or `-Y`
+(trusted X11 forwarding) option when launching the SSH client:
+
+```bash
+ssh -X <username>@<hostname>.csc.fi
+```
+
+For more information about the X11 forwarding options, run `man ssh` in the
+terminal.
+
+## Generating SSH keys
```

Comment:
Perhaps we should link to the ssh-keys.md page as well?

Reply:
9790dfd1